umbraco-cloud/set-up/going-live.md

```diff
@@ -16,7 +16,8 @@ When you create a project on Umbraco Cloud, the generated project URL is based o
 
 {% hint style="info" %}
 Before adding a hostname, you need to update your DNS host domain registrar DNS entries to resolve to `umbraco.io`. We recommend setting a CNAME record for your site using the `dns.umbraco.io` Umbraco Cloud DNS record.
-{% endhint %}
+{% endhint %} 
+You can read more about how to do this under [Manage Hostnames](project-settings/manage-hostnames/).
```

Comment:
changed the text to be inside the info block as it makes more sense to be in the same paragraph where it mentions about DNS settings
```suggestion
You can read more about how to do this under [Manage Hostnames](project-settings/manage-hostnames/).
{% endhint %} 
```